examples/llava/clip.h

```diff
@@ -60,6 +60,8 @@ CLIP_API struct clip_image_f32 * clip_image_f32_init();
 
 CLIP_API void clip_image_u8_free (struct clip_image_u8  * img);
 CLIP_API void clip_image_f32_free(struct clip_image_f32 * img);
+CLIP_API void clip_image_u8_batch_free (struct clip_image_u8  * data);
+CLIP_API void clip_image_f32_batch_free(struct clip_image_f32 * data);
```

Comment:
Wouldn't it be better to change these to:

```c
CLIP_API void clip_image_u8_batch_free (struct clip_image_u8_batch  * batch) [
    if (batch.size > 0) {
        delete[] batch.data;
    }
    batch.size = 0;
}
```

Reply:
Agreed, I changed it and retested.